src/DynamoPublish/DynamoPublishExtension.cs

```diff
@@ -112,29 +119,111 @@ private MenuItem GenerateMenuItem()
             var isEnabled = loadedParams.CurrentWorkspaceModel is HomeWorkspaceModel && startupParams.AuthProvider != null;
             item.IsEnabled = isEnabled;
 
-            item.Click += (sender, args) =>
+            // Open the configuration file using the dll location.
+            var config = ConfigurationManager.OpenExeConfiguration(typeof(PublishModel).Assembly.Location);
+            // Get the appSettings section.
+            var appSettings = (AppSettingsSection)config.GetSection("appSettings");
```

Comment:
@pbidenko But your cast would throw an exception before that! I'd recommend always using `as`. It's faster and prevents exceptions.